### ISSUE TITLE
OctahedralGaussianGrid not defined

### ISSUE THREAD

**Issue by matteodefelice** [milankl/SpeedyWeather.jl#168]
Installed SpeedyWeather.jl on Julia 1.6.4 for Windows. After executing, as in the example, ` run_speedy(n_days=30, trunc=63, Grid=OctahedralGaussianGrid, model=:shallowwater, output=true)` I get:

```
ERROR: UndefVarError: OctahedralGaussianGrid not defined
Stacktrace:
 [1] top-level scope
   @ REPL[3]:1
```

**Comment by milankl** [comment]
Which version is installed? We have restricted to julia ^1.7 at some point I think, so you probably don’t have v0.4. Please use a newer version of Julia

**Comment by matteodefelice** [comment]
I have uninstalled my Julia 1.6.4 and installed the 1.8.2 but still I get the version 0.3, getting the same error.

**Comment by milankl** [comment]
Yes, grids are only implemented with v0.4, check Julia’s package manager to see how you can identify which package is limiting the update or how you can force upgrade to v0.4

**Comment by milankl** [comment]
Closing this as in v0.4 `OctahedralGaussianGrid` is defined
```julia
julia> using SpeedyWeather

julia> OctahedralGaussianGrid
OctahedralGaussianGrid
```
feel free to raise a new issue for other questions.
